bin/pocs_shell

```diff
@@ -74,6 +74,17 @@ class PocsShell(Cmd):
 
         return self.pocs.is_safe()
 
+    def do_display_config(self, *arg):
+        """ Display the config file for POCS """
+        try:
+            if self.pocs.config=={}:
```

Comment:
Same here where you just want to check `if self.pocs.config:`, but I think the point James was making was that if you have not run `setup_pocs` then `self.pocs` will be `None` and the `self.pocs.config` will generate an error. So you first need to check for `pocs` and then check for `config`, i.e. `if self.pocs and self.pocs.config:`